src/api/helpers.ts

```diff
@@ -0,0 +1,18 @@
+export function getCookie(name: string): string | null {
```

Comment:
```suggestion
export function getCookiesClientSide(name: string): string | null {
```
It is more understandable this